component System_error3 {
	def main(arg: Array<string>) -> int {
		foo();
		return 0;
	}
	def foo() {
		System.error("SystemError", "with a message");
	}
}
